src/testing/mocks.rs

```diff
@@ -363,7 +363,7 @@ pub type MockBackend<'a> = MockBackendWithHeight<'a, 5>;
 pub type MockNetwork<'a> = MockNetworkWithHeight<'a, 5>;
 pub type MockSystem = MockSystemWithHeight<5>;
 
-#[cfg(test)]
+#[cfg(all(test, feature = "slow-tests"))]
```

Comment:
This disables pretty much all the tests in the normal build, I think. Can we be a bit more fine-grained and annotate specific tests in `tests.rs` with `slow-tests`? This will have the added benefit of allowing downstream crates to instantiate the generic test suite in both builds, and having the slow ones only run in the slow build, with something like
```toml
[features]
slow-tests = ["seahorse/slow-tests"]
```